dts/arm/nxp/nxp_s32z27x_rtu1_r52.dtsi

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <dt-bindings/clock/nxp_s32z2_clock.h>
```

Comment:
no need to include it here since this file already includes `<arm/nxp/nxp_s32z27x_r52.dtsi>`

Reply:
oh, right. removed that.